src/containers/shared/components/Transaction/AMM/AMMDeposit/parser.tsx

```diff
@@ -0,0 +1,18 @@
+import { getAMMAccountID, findAssetAmount } from '../../../../MetaParser'
+import { XRP_BASE } from '../../../../transactionUtils'
+
+export function parser(tx: any, meta: any) {
+  const amount = tx.Asset
+  const amount2 = tx.Asset2
+  const accountID = getAMMAccountID(meta)
+  const fee = tx.Fee / XRP_BASE
```

Comment:
Fee is already shown on the right and shouldnt need to come from the parser.

Reply:
it exists in a different component so we'd have to pass it in somehow either way.